qa/L0_trace/test.sh

```diff
@@ -239,9 +239,6 @@ assert_curl_failure "trace_file updated through network protocol expects an erro
 if [ `grep -c "\"error\":\"trace file location can not be updated through network protocol\"" ./curl.out` != "1" ]; then
     RET=1
 fi
-if [ `grep -c "\"trace_mode\":\"triton\"" ./curl.out` != "1" ]; then
```

Comment:
can we please add a `TODO` or something, to indicate that when `trace_file` update is back, we need to test `trace_mode` as well.

Reply:
Don't we have other places in the test that test the returned mode is correct? Isn't the `trace_file` update / error check more standalone?